libraries/debian_helpers.rb

```diff
@@ -6,27 +6,26 @@ module Helpers
     module Debian
       def enable_auto_updates_debian
         package 'debconf-utils' do
-          action :install
+          action :upgrade
         end
         package 'unattended-upgrades' do
-          response_file lazy { 'unattended-upgrades.seed.erb' }
-          action :install
+          response_file 'unattended-upgrades.seed.erb'
+          action :upgrade
         end
-        unattended_upgrade_file = '/etc/apt/apt.conf.d/50unattended-upgrades'
+        unattended_upgrades_file = '/etc/apt/apt.conf.d/50unattended-upgrades'
         execute 'rackspace_allowed_origins' do
-          command "perl -pi -e 's/(Allowed-Origins\s*{)/$1\n\t\"serveragent main\";\n\t\"cloudmonitoring main\";/;' #{unattended_upgrade_file}"
+          command "perl -pi -e 's/(Allowed-Origins\s*{)/$1\n\t\"serveragent main\";\n\t\"cloudmonitoring main\";/;' #{unattended_upgrades_file}"
           only_if { rackspace? }
           action :run
         end
         # New format(Currently only Deb 7)
         execute 'rackspace_unattended_upgrade' do
-          command "perl -pi -e 's/(Unattended-Upgrade::Origins-Pattern\s*{)/$1\n\t\"o=serveragent,a=main\";\n\t\"o=cloudmonitoring,a=main\";/;' #{unattended_upgrade_file}"
+          command "perl -pi -e 's/(Unattended-Upgrade::Origins-Pattern\s*{)/$1\n\t\"o=serveragent,a=main\";\n\t\"o=cloudmonitoring,a=main\";/;' #{unattended_upgrades_file}"
           only_if { rackspace? }
           action :run
         end
         execute 'enable_auto_updates' do
           command 'echo unattended-upgrades unattended-upgrades/enable_auto_updates boolean true | debconf-set-selections'
-          only_if 'debconf-get-selections | grep unattended-upgrades | grep boolean | grep false'
```

Comment:
@martinb3 I don't see it as a "problem" per se, however, I do share the same sentiments regarding the updating of resources that are not actually changing anything, in most cases. I removed it for now to help address an issue with testing on Ubuntu 14.04. It seemed that when this resource had the guard, the output from `debconf-get-selections | grep unattended-upgrades | grep boolean | grep false` only returned an empty string. Thus, even though it didn't state false in the output it still didn't match the guard so the resource wouldn't run to fix the config and the integration tests for unattended-upgrades would fail. I could probably change the guard to capture both `false` and `""` then it would work properly but because I didn't necessarily see it as a "problem" I chose the easier approach and removed the guard.. I would be happy to work on that point though and submit a PR with the updated guard if you would like? I am open to suggestions there. Thanks!